### PR TITLE
gh-100829: Show the Python Launcher version

### PR DESCRIPTION
**Pull request by oleksis**
- https://github.com/python/cpython/issues/100829#issue-1524155822

* Issue: gh-100829

### CODE REVIEW

File: PC/launcher2.c [hunk 1]
```diff
@@ -647,6 +650,8 @@ parseCommandLine(SearchInfo *search)
                 search->help = true;
                 // Do not update restOfCmdLine so that we trigger the help
                 // message from whichever interpreter we select
+            } else if (STARTSWITH(L"0v") || STARTSWITH(L"-version-launcher")) {
```
Comment (zooba) [inline PC/launcher2.c]: These should be `MATCHES` checks, as they don't have any other parameters.

Comment (oleksis) [reply PC/launcher2.c]: Thanks for the comments

**Comment by oleksis** [comment]
@zooba i update the changes for `py` launcher show the version of the aplication binary . Testing the changes with this [versionLauncher.patch](https://github.com/oleksis/pylauncher/blob/master/patches/versionLauncher.patch)

Any advice, welcome!

**Comment by zooba** [comment]
There's still discussion (on the issue) to resolve about what this is for. I don't believe it's worth adding an extra option solely for users to run themselves, which is what the current PR does.

**Comment by terryjreedy** [comment]
The issue has been closed, so I think this PR should be also.

**Pull request closed without merging**